dynatrace/build.gradle

```diff
@@ -20,14 +20,14 @@ plugins {
 }
 
 group = "com.dynatrace.opentelemetry.metric"
-version = "0.3.0-alpha"
+version = "0.4.0-alpha"
 
 repositories {
     mavenCentral()
     jcenter()
```

Comment:
JCenter will stop serving packages after Febuary 1st, 2022. Should we remove it now?

Reply:
I'm not really familiar with this, but sounds like a good idea then?

Reply:
Ok, removed.